explorer/mempool.go

```diff
@@ -208,11 +194,24 @@ func (exp *explorerUI) storeMempoolInfo() (lastBlockHash string, lastBlock int64
 	return
 }
 
-func voteForLastBlock(blockHash, validationHash string) bool {
-	if blockHash != "" && validationHash != blockHash {
-		return false
+func addVoteIndex(tx *MempoolTx, txindexes map[int64]map[string]int) {
```

Comment:
Despite being non-exported, it would be helpful to have a description of this function to the intent is clear.  Looks like it is updating `tx.VoteInfo.MempoolTicketIndex` and/or the txindexes map.  Actually, perhaps it should take a `*VoteInfo` instead of a `*MempoolTx`.